elife.cfg

```diff
@@ -3,6 +3,8 @@ env: dev
 #upload_path uploads
 # lax--ci needs it's own app.cfg file
 cache_path: /home/elife
+cdn_iiif: https://iiif.elifesciences.org/lax:
```

Comment:
I think the build is failing because of this default, it should be `https://prod--cdn-iiif.elifesciences.org/lax:`. Can you update it?

Reply:
Yes, sorry for the delay.